### PR TITLE
Face C# quickstart

### PR DESCRIPTION
**Pull request by wiazur**
Please use `[API Name] Brief description` as title.

## API Name (Kind)
`Face`

## Purpose
Describe the intention of the changes being proposed.
What problem does it solve or functionality does it add?
Adding a new quickstart solution.

### CODE REVIEW

File: documentation-samples/quickstarts/Face/FaceQuickstart.sln [hunk 1]
```diff
@@ -0,0 +1,25 @@
+﻿
```
Comment (yangyuan) [inline documentation-samples/quickstarts/Face/FaceQuickstart.sln]: Are you sure you want to check-in sln and csproj files?

File: documentation-samples/quickstarts/Face/FaceQuickstart/FaceQuickstart.csproj [hunk 2]
```diff
@@ -0,0 +1,70 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (yangyuan) [inline documentation-samples/quickstarts/Face/FaceQuickstart/FaceQuickstart.csproj]: if you have to check-in csproj (which I believe you don't). Please consider to use .NET core csproj format.

File: documentation-samples/quickstarts/Face/FaceQuickstart/obj/Debug/FaceQuickstart.csproj.CoreCompileInputs.cache [hunk 3]
```diff
@@ -0,0 +1 @@
+45ff8c8a0bab805f137dec2bdd9e8882f5a74aa0
```
Comment (yangyuan) [inline documentation-samples/quickstarts/Face/FaceQuickstart/obj/Debug/FaceQuickstart.csproj.CoreCompileInputs.cache]: Please remove all the support files.
Such as files under obj/bin, packages.

**Comment by wiazur** [comment]
@yangyuan I deleted all the files you suggested. But now it says there is a conflict with the README file. Not sure why, I did not change this file in my latest commit.

**Comment by yangyuan** [comment]
Hi @wiazur 

I noticed you modified the files. But you should also "resolve" the comments in PR so that I can know you consider the issue has been fixed.

I recommend to remove sln file, and change csproj file to new format. It should be something like below.

```
<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <OutputType>Exe</OutputType>
    <TargetFrameworks>netcoreapp2.0;net461;netstandard1.4</TargetFrameworks>
  </PropertyGroup>

</Project>
```

File: documentation-samples/quickstarts/Face/FaceQuickstart/FaceQuickstart.csproj (hunk 2 above)
Comment (wiazur) [reply documentation-samples/quickstarts/Face/FaceQuickstart/FaceQuickstart.csproj]: Thanks @yangyuan  I deleted it.

File: documentation-samples/quickstarts/Face/FaceQuickstart.sln (hunk 1 above)
Comment (wiazur) [reply documentation-samples/quickstarts/Face/FaceQuickstart.sln]: I deleted them, thanks.

**Comment by wiazur** [comment]
Thanks Yuan,

I deleted the csproj and sln files. Also resolved the conversations.

https://github.com/Azure-Samples/cognitive-services-dotnet-sdk-samples/pull/119/commits/4e00b4d4105f070d219fcb76d211fdeb4e00b9f4

If there is anything else I need to do, let me know.

Thanks much,
Winona

**Pull request merged**